packages/react/src/components/TooltipDefinition/TooltipDefinition-story.js

```diff
@@ -15,25 +15,52 @@ const directions = {
   'Bottom (bottom)': 'bottom',
   'Top (top)': 'top',
 };
-
 const alignments = {
   'Start (start)': 'start',
   'Center (center)': 'center',
   'End (end)': 'end',
 };
-
-const props = () => ({
-  direction: select('Tooltip direction (direction)', directions, 'bottom'),
-  align: select(
-    'Tooltip alignment to trigger button (align)',
-    alignments,
-    'start'
+const triggerElements = {
+  button: 'button',
+  div: 'div',
+  span: 'span',
+};
+const triggerElementMap = {
+  button: undefined,
+  div: ({ id, children, ...props }) => (
+    <div aria-describedby={id} {...props}>
```

Comment:
`aria-describedby` is intended for interactive elements and will be ignored by NVDA, VoiceOver when applied to divs and spans. This would need an interactive role of some kind to function as intended.  The implicit role we get with `button` covered us here, but now that we're allowing non-semantic markup maybe a `role="tooltip"` would make up the difference. I'd need to test further once the changes were made.

https://developer.paciellogroup.com/blog/2017/07/short-note-on-aria-label-aria-labelledby-and-aria-describedby/
https://www.w3.org/TR/using-aria/#label-support

Also worth noting is when using `aria-describedby` in Internet Explorer. MS requires the referenced element (via ID) be a **Microsoft Accessible Element**. Essentially it needs to have proper roles etc. 

_NOTE: this comment also applies to every other instance of this pattern throughout the rest of the PR. I'll keep it to one comment to avoid clutter/noise._